src/modules/languages/python.nix

```diff
@@ -39,6 +42,21 @@ let
     follows = [ "nixpkgs" ];
   };
 
+  pth_file =
+    let
+      exec_content = ''
+        ld_library_path_var = "DYLD_LIBRARY_PATH" if sys.platform == "darwin" else "LD_LIBRARY_PATH"
+        ld_library_path = os.environ.get(ld_library_path_var)
+        ld_library_path_prefix = os.environ.get("DEVENV_LD_LIBRARY_PATH_PREFIX")
+        if ld_library_path and ld_library_path_prefix:
+            if ld_library_path == ld_library_path_prefix:
+                del os.environ[ld_library_path_var]
+            else:
+                os.environ[ld_library_path_var] = ld_library_path.removeprefix(ld_library_path_prefix + ":")
+      '';
+    in
+    pkgs.writeText "devenv.pth" ''import os; exec("""${builtins.replaceStrings [ "\n" ] [ "\\n" ] exec_content}""")'';
```

Comment:
Does this need `exec`? I thought it might be possible to be executed in pth as-is.

Reply:
Pth is executed line by line, so this is the only way to use multi-line constructs like `if` conditions